tests/core/daemon/test_daemon.py

```diff
@@ -51,6 +51,20 @@ def id(self) -> str:
         return f"{self.route}: {self.description}"
 
 
+@dataclass
+class RouteCaseStatus:
```

Comment:
I expect `Case` to be at the end, but perhaps I'm misreading this somehow.

Reply:
adjusted